production-marathon/marathon.env

```diff
@@ -1,16 +1,17 @@
 # The following properties MUST be configured before the Job Service can be deployed.
 
-export DOCKER_REGISTRY=NOT_SET_QA_index.docker.io/jobservice
+export DOCKER_REGISTRY=NOT_SET_DOCKER_REGISTRY
```

Comment:
The contents of this environment file could possibly change today depending on any further perl command changes.